app.py

```diff
@@ -1,10 +1,11 @@
 from bottle import request, route, run, template, post, TEMPLATE_PATH, abort, redirect
 from marshmallow import Schema, fields
-import dbfunctions
+from dbfunctions import Wikidb
 
 TEMPLATE_PATH.append('templates')
 DB_PATH = './wiki.db'
 
+db = Wikidb(db_path=DB_PATH)
```

Comment:
> I figure failing sooner is better than later
That's certainly a good thing to keep in mind.

I've had issues with other libraries assuming certain things exist on the system you are importing the module from.  For example:
```python
import os
if not os.path.exists('c:\\my\\sweet\\app.exe'):
    raise Exception("no app!")

def some_really_cool_function_that_has_nothing_do_with_app_exe():
    pass
```

I have to go through a lot of effort in order to be able to use `some_really_cool_function_that_has_nothing_do_with_app_exe()`.  Why should I?  Why couldn't I simply do this in my code:
```python
import some_module
some_module.some_really_cool_function_that_has_nothing_do_with_app_exe()
```

Like I said, it's not "wrong" to execute code during import.  I don't like to see it because it may cause unintended lameness when someone attempts to import your module.

In this particular case, *not* creating the DB on import may also make writing tests a bit easier (although I haven't actually looked at it).  I thought about that during project night, but I forgot which particularity problem it would have solved ;)